### ISSUE TITLE
On the way scheduler calculate the number of sequences

### ISSUE THREAD

**Issue by LinPoly** [vllm-project/vllm#257]
In version 0.1.1, the scheduler calculates the number of sequences it schedules like the following:
```python
num_new_seqs = seq_group.num_seqs(status=SequenceStatus.SWAPPED)
num_curr_seqs = len(self.running)
if num_curr_seqs + num_new_seqs > self.scheduler_config.max_num_seqs:
    break
```
`num_curr_seqs` is the number of sequence groups while `num_new_seqs` stands for the number of sequences in the new sequence group, I think it cannot form a meaningful metric by adding them together.

**Comment by WoosukKwon** [comment]
@LinPoly You're right. This indeed seems like a bug. Thanks for reporting this and please keep exploring vLLM!
